src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -1179,14 +1180,27 @@ var TorrentsTable = new Class({
                 break;
         }
 
-        if (categoryHash == CATEGORIES_ALL)
-            return true;
-
-        if (categoryHash == CATEGORIES_UNCATEGORIZED && row['full_data'].category.length === 0)
-            return true;
+        switch (isNaN(categoryHash) || parseInt(categoryHash)) {
```

Comment:
this is hard to comprehend... is it possible to write to following?
```js
switch (parseInt(categoryHash)) {
    case NaN:
        break;
    // others
}
```

Reply:
I don’t believe so, which is what lead me to this funky conditional. I’ll double check though.

Reply:
Teste, this doesn't work. `NaN` is pretty counterintuitive in JS. For example, this evaluates to false: `parseInt("sdfsdfsdf") === NaN`.

Reply:
I moved the `isNaN` check up so the code is more readable